res/controllers/Electrix-Tweaker-scripts.js

```diff
@@ -879,20 +880,24 @@ ElectrixTweaker.playButton = function (channel, control, value, status, group) {
     }
 }
 ElectrixTweaker.playButtonLED = function (value, group, control) {
-    if (engine.getValue(group, 'play')) {
-        if (
-            (control != 'playposition') // do not spam MIDI signals with each update in playposition while playing
-            && (
-                (! ElectrixTweaker.hotcuesPressed[group])
-                || ElectrixTweaker.playPressedWhileCueJuggling[group]
-            )
-        ) {
-            midi.sendShortMsg(0x90, ElectrixTweaker.buttons[group]['play'], ElectrixTweaker.colorCodes['green'])
-        }
-    } else if (engine.getValue(group, 'track_samples')) {
-        midi.sendShortMsg(0x90, ElectrixTweaker.buttons[group]['play'], ElectrixTweaker.colorCodes['red'])
+    if (control == 'cue_indicator' && value == 1) {
```

Comment:
Generally, yes, but I was keeping it internally consistent with the rest of the script. I plan on rewriting the script (with better style) after finishing the P32 mapping to continue refining the JS library I'm working on with the P32 mapping.